editor.planx.uk/src/pages/FlowEditor/components/PreviewBrowser.tsx

```diff
@@ -68,7 +72,7 @@ const DebugConsole = () => {
   );
 };
 
-function PublishChangeItem(props: any) {
+function AlteredNodeItem(props: any) {
```

Comment:
I'm aware there's some lingering `any`'s around this file that I haven't cleaned up - one on my radar to come back to, want to keep up momentum with logic changes first !